app/src/main/java/org/p2p/wallet/utils/DecimalFormatUtil.kt

```diff
@@ -0,0 +1,21 @@
+package org.p2p.wallet.utils
+
+import android.icu.text.DecimalFormat
+import android.icu.text.DecimalFormatSymbols
+import java.util.Locale
+
+private const val DECIMAL_FORMAT = "###,###."
+
+object DecimalFormatUtil {
```

Comment:
Utils are not hype anymore, old man! `*Ext` or `DecimalFormatter`

Reply:
Nice point, zoomer